<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="ru-ru">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e236f97a354e379df194e142596e08daa70d35f4</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.IdentityModel.Selectors.ServiceBusyException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">79a3c13ebe4274c8a5c1aaa9f6c81721498a06a9</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">bf947afa611911f9a5230901750a8c4dd2d7ef15</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>Exception class to indicate that the <ph id="ph1">[!INCLUDE[infocard](~/add/includes/infocard-md.md)]</ph> service is busy processing other requests.</source>
          <target state="translated">Класс исключений, чтобы указать, что <ph id="ph1">[!INCLUDE[infocard](~/add/includes/infocard-md.md)]</ph> служба занята обработкой других запросов.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>A ServiceBusyException exception is thrown to indicate that the CardSpace service is busy processing other requests.</source>
          <target state="translated">ServiceBusyException-исключение вызывается, чтобы указать, что служба CardSpace занята обработкой других запросов.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>CardSpace does not queue requests, and can service only one request at a time.</source>
          <target state="translated">CardSpace не ставит запросы в очередь и может обслуживать только один запрос единовременно.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>Determine whether another instance of CardSpace is running.</source>
          <target state="translated">Определите, выполняется ли другой экземпляр CardSpace.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>If there is another instance running, end it by stopping the icardagt.exe process from Task Manager.</source>
          <target state="translated">Если выполняется, завершите его, остановив процесс icardagt.exe из диспетчера задач.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> класса.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the specified error message.</source>
          <target state="translated">Инициализирует новый экземпляр класса <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> класса с указанным сообщением об ошибке.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error.</source>
          <target state="translated">Этот конструктор инициализирует &lt;xref:System.Exception.Message%2A&gt;свойств нового экземпляра системным сообщением, содержащим описание ошибки.&lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>The content of the message should also be understandable to the user.</source>
          <target state="translated">Содержимое сообщения должно быть понятно пользователю.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>The message that describes the error.</source>
          <target state="translated">Сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialization information and streaming context specified.</source>
          <target state="translated">Инициализирует новый экземпляр класса <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> класс со сведениями о сериализации и контекстом потоковой передачи указано.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Данный конструктор вызывается в процессе десериализации для воссоздания объекта исключения, переданного в потоке.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>The <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object that holds the serialized object data about the exception that is being thrown.</source>
          <target state="translated"><bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> Объект, содержащий сериализованные данные объекта о вызываемом исключении.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>The &lt;xref:System.Runtime.Serialization.StreamingContext&gt; object that contains contextual information about the source or destination.</source>
          <target state="translated">&lt;xref:System.Runtime.Serialization.StreamingContext&gt;Объект, содержащий контекстные сведения об источнике или назначении.&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>Initializes an instance of the <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Инициализирует новый экземпляр класса <bpt id="p1">&lt;xref href="System.IdentityModel.Selectors.ServiceBusyException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> класса с указанным сообщением об ошибке и ссылкой на внутреннее исключение, вызвавшее данное исключение.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Исключение, созданное как прямой результат предыдущего исключения, должно содержать ссылку на предыдущее исключение в &lt;xref:System.Exception.InnerException%2A&gt;свойство.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">&lt;xref:System.Exception.InnerException%2A&gt;Свойство возвращает то же значение, которое передается конструктору, или <ph id="ph1">`null`</ph> Если &lt;xref:System.Exception.InnerException%2A&gt;свойства не предоставляет конструктору значение внутреннего исключения.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>The message that describes the error.</source>
          <target state="translated">Сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Исключение, которое стало причиной текущего исключения.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>